int32 seqence_number
float32 ping_pong_time
float32 offset
